### ISSUE TITLE
Consider adding an option to save the screenshot on the server as a file rather than download it on the client side

### ISSUE THREAD

**Issue by daattali** [daattali/shinyscreenshot#3]
This was suggested to me by email and I think it would be a nice feature. Need to look into this and see if it's possible.

**Comment by thierryzorn** [comment]
The idea could be to save it on the server, or in a local rep, for example the tempdir(), but without any prompt, and, optionnally an indicator wether the saving has been successful

**Comment by daattali** [comment]
Yep I have an idea of what the behaviour should look like, I just need to spend some time looking how to actually get that done. Or I would be happy to welcome a PR.

**Comment by daattali** [comment]
Coincidentally, 3 different people emailed me requesting this feature within 2 weeks!

I've been thinking about the best user behaviour (and asked @oganm) and I think I'll add two parameters: `download=TRUE/FALSE` would decide if the end user gets the image downloaded, and `server_path=NULL/string` when a string is provided, the image will be saved to this location in the server

**Comment by thierryzorn** [comment]
A way to proceed could be either to store it transparently, without validating by clicking, in the same way you write data with write_rds() or  write_csv().
The storing path would be in the path parameter.

**Comment by daattali** [comment]
Sorry, I don't understand what you mean, could you rephrase?

To store on the server, you do have tons specify a location. To store on the client side, you *cannot* specify the location in shiny, it must be done by the end user in their browser.

**Comment by thierryzorn** [comment]
Yes, sorry you're right. To store on the server, you just have to specify the name, and it stores it on the distant tempdir(), where you can retrieve it .

**Comment by daattali** [comment]
As mentioned in [this comment](https://github.com/daattali/shinyscreenshot/issues/3#issuecomment-731632863), I think I'll expose a parameter that will allow you to choose the path on the server rather than go to a tempdir.

**Comment by daattali** [comment]
This has been implemented. There are now `download=TRUE/FALSE` and `server_dir="path/to/save"` parameters
